core/src/table/table.tsx

```diff
@@ -64,6 +64,11 @@ export interface TableProps<R> {
 	 * scrolling ancestor (i.e. one with "auto" or "scroll" overflow).
 	 */
 	fixed?: boolean;
+	/**
+	 *	when you want to full 100% width of table
+	 */
```

Comment:
Should use https://github.com/moaijs/moai/pull/91#discussion_r601736274 and also should not have that tab